nixos/modules/services/networking/prosody.nix

```diff
@@ -541,7 +542,9 @@ let
         ${lib.concatStringsSep "\n  " (
           lib.mapAttrsToList (name: val: optionalString val "${toLua name};") cfg.modules
         )}
-        ${lib.concatStringsSep "\n" (map (x: "${toLua x};") cfg.package.communityModules)}
+        ${lib.concatStringsSep "\n" (
+          map (x: "${toLua x};") (lib.filter (n: n != "muc_notifications") cfg.package.communityModules)
```

Comment:
You can replace the composition of filter, map with just a fold instead of doing multiple iterations. It would also be better to prep for a list of modules that might be excluded in the future. 

I would also defer to @SuperSandro2000 who appears to actually be using this module (I moved to Ejabberd a few years back).

Reply:
Fixed, thanks for review!
@SuperSandro2000, could you please have look one more time?